.github/workflows/Nightly.yml

```
@@ -0,0 +1,100 @@
+name: Nightly-Nightly-Check
+
+on:
+  schedule:
+    # nightly tests
+    - cron: "0 0 * * *"
+  # Be able to run manually
+  workflow_dispatch:
+
+jobs:
+  test:
+    name: Test on ${{ matrix.cfg.os }}, Python ${{ matrix.cfg.python-version }}, OpenMM ${{ matrix.cfg.openmm }}
```

What about this, but on `.github/workflows/CI.yaml`?

```suggestion
    if: (github.event_name == 'schedule'  && matrix.cfg.nightly) || github.event_name != 'schedule' 
    name: Test on ${{ matrix.cfg.os }}, Python ${{ matrix.cfg.python-version }}, OpenMM ${{ matrix.cfg.openmm }}
```

You'll need a `nightly` key on the relevant matrix entries.

---

The individual steps will need to check against `if: matrix.cfg.nightly` only.

---

I cut those when at first I was dropping nightly from running on PRs